### PR TITLE
Updated All Ability Descriptions

### PR DESCRIPTION
**Pull request by ezrn**
-Added new Effect and Stat color to every ability
-Reworded to make them understandable
-Made everything much more consistent across the board

### CODE REVIEW

File: champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/knight/passives/Deflection.java [hunk 1]
```diff
@@ -46,13 +46,11 @@ public String getName() {
...
 
         return new String[]{
-                "Prepare to deflect incoming attacks",
-                "You gain <val>1</val> charge every <val>" + timeBetweenCharges + "</val> seconds.",
+                "You gain <stat>1</stat> charge every <stat>" + timeBetweenCharges + "</stat> seconds.",
                 "You can store a maximum of <val>" + (level) + "</val> charges",
                 "",
                 "When attacked, the damage you take is",
-                "reduced by the number of your charges",
-                "and your charges are reset to 0."
```
Comment (StudentAlleg) [inline champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/knight/passives/Deflection.java:55]: Why was this removed?

File: champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/knight/passives/Swordsmanship.java [hunk 2]
```diff
@@ -48,15 +48,13 @@ public String getName() {
...
         return new String[]{
-                "Prepare a powerful sword attack,",
-                "You gain 1 charge every <val>" + timeBetweenCharges + "</val> seconds.",
-                "You can store a maximum of <val>" + (level) + "</val> charges",
+                "You gain 1 charge every <stat>" + timeBetweenCharges + "</stat> seconds,",
+                "storing up to a maximum of <val>" + (level) + "</val> charges",
                 "",
                 "When you attack, your damage is",
-                "increased by <val>" + damagePerCharge + "</val> for each charge you have",
-                "and then your charges are reset to 0.",
```
Comment (StudentAlleg) [inline champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/knight/passives/Swordsmanship.java:57]: Why was this removed?

File: champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/paladin/axe/Boulder.java [hunk 3]
```diff
@@ -50,8 +50,10 @@ public String getName() {
...
     public String[] getDescription(int level) {
         return new String[] {
-                "Throw a boulder forward that, upon",
-                "contact, deals <val>" + getDamage(level) + "</val> damage to all nearby",
+                "Right click with an Axe to activate",
+                "",
+                "Throw a boulder forward that",
+                "deals <val>" + getDamage(level) + "</val> damage to all nearby",
                 "enemies and heals surrounding allies for",
                 "<val>" + getHeal(level) + "</val> health in a radius of <val>" + getRadius(level) + "</val> blocks.",
```
Comment (StudentAlleg) [inline champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/paladin/axe/Boulder.java:58]: radius of boulder?

**Review by StudentAlleg — COMMENTED** [review]
48/96 files viewed.

File: champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/knight/passives/Deflection.java (hunk 1 above)
Comment (ezrn) [reply champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/knight/passives/Deflection.java:55]: redundant information that is implied by the rest of the skill description

File: champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/paladin/axe/Boulder.java (hunk 3 above)
Comment (Mykindos) [reply champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/paladin/axe/Boulder.java:58]: Boulder is still a WIP

**Review by Mykindos — APPROVED** [review]
